pants3

```diff
@@ -4,10 +4,34 @@
 
 # This bootstrap script invokes Pants using a Python 3 interpreter.
 
-# Use Py3 under-the-hood
-export PANTS_USE_PYTHON3=true
+# Defaults to allowing either Python 3.6 or 3.7, and allows the user to further constrain to a specific version
```

Comment:
Per the comment in `virtualenv` this could just use `PY`, defaulting it to `python3.6` if unset. Then invoke `$PY` with a script to spit out the major.minor version (I find this takes 50ms for 3.6 and 15ms for 3.7) and:
```
export PANTS_PYTHON_SETUP_INTERPRETER_CONSTRAINTS="['CPython>=$major_minor.*']"
```
You could potentially make the logic slightly more complex and skip the `$PY` invoke for the default case since you know the answer already.